### PR TITLE
Rename ophan file to enable testing

### PR DESCRIPTION
**Pull request by rupertbates**
## What are you doing in this PR?
This PR renames the ophan module to trackingOphan. 

The reason for doing this is that when it was named ophan it was not possible to test any functions in it because it is getting mocked by this code here: https://github.com/guardian/support-frontend/blob/08643c2d5aa3875a5934db82758c2d3f73852a06/support-frontend/jestSetup.js#L6-L10

Strangely, changing the module name to eg. ophanTracking still results in the same issue, the name needs to start with something other than ophan to work.

If anyone has a better solution for this please chip in!

### CODE REVIEW

File: support-frontend/assets/helpers/tracking/__tests__/ophanTest.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import type { Participations } from '../../abTests/abtest';
+import type { OphanABPayload } from '../trackingOphan';
+import { buildOphanPayload } from '../trackingOphan';
```
Comment (rupertbates) [inline support-frontend/assets/helpers/tracking/__tests__/ophanTest.ts:3]: We can now test functions in the trackingOphan module

**Review by jamesgorrie — APPROVED** [review]
This is really odd behavior from jest. 

`Jest.mock('ophan')` seems to mock `ophan` in the `node_modules` but also when importing the local `./ophan`.

Even more strangely, on investigation, it seems to also match a module called `ooooooooophan` or pretty much anything similar. 

This solution works - but it would be good to find out why jest is working in this way and if we can make it more explicit so this doesn't sting someone in the future.

The docs don't say that it should work this way, but also doesn't say it shouldn't... 🤔

**Review by jamesgorrie — COMMENTED** [review]
Actually - re-looking at this - [I think we might be able to have this set more explicitly here](https://github.com/guardian/support-frontend/blob/main/support-frontend/package.json#L63) - seems weird that we're pattern matching

**Comment by rupertbates** [comment]
> Actually - re-looking at this - [I think we might be able to have this set more explicitly here](https://github.com/guardian/support-frontend/blob/main/support-frontend/package.json#L63) - seems weird that we're pattern matching

Oh yes good spot, that is weird

**Comment by rupertbates** [comment]
> Actually - re-looking at this - [I think we might be able to have this set more explicitly here](https://github.com/guardian/support-frontend/blob/main/support-frontend/package.json#L63) - seems weird that we're pattern matching

I've tried making this just ophan with no pattern match and the behaviour is still the same. I'm going to go with this solution for now as it works and allows me to move forward.

**Pull request merged**